### PR TITLE
collection period for V2 to v1 migration is no longer hard coded.

### PR DESCRIPTION
**Pull request by abroodie**

### CODE REVIEW

File: src/SFA.DAS.Payments.Migration/Program.cs [hunk 1]
```diff
@@ -196,6 +196,15 @@ private static LegacyPeriodModel CreateTrigger()
 
         private static async Task ProcessV1Payments()
         {
+            await Log("");
+            await Log("Please enter the collection period: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12");
```
Comment (carlbm) [inline src/SFA.DAS.Payments.Migration/Program.cs]: Is it worth including 13 and 14?

File: src/SFA.DAS.Payments.Migration/Program.cs [hunk 2]
```diff
@@ -213,7 +222,7 @@ private static async Task ProcessV1Payments()
                 do
                 {
                     // Load from v2
-                    paymentsAndEarnings = (await v2Connection.QueryAsync<V2PaymentAndEarning>(V2Sql.PaymentsAndEarnings,
+                    paymentsAndEarnings = (await v2Connection.QueryAsync<V2PaymentAndEarning>(string.Format(V2Sql.PaymentsAndEarnings, collectionPeriod),
```
Comment (carlbm) [inline src/SFA.DAS.Payments.Migration/Program.cs]: Dapper allows you to pass named parameters. The pattern is to use `@name` in the sql and then pass an anonymous object with a property `Name`

Comment (abroodie) [reply src/SFA.DAS.Payments.Migration/Program.cs]: done

File: src/SFA.DAS.Payments.Migration/Program.cs (hunk 1 above)
Comment (abroodie) [reply src/SFA.DAS.Payments.Migration/Program.cs]: done

**Pull request merged**